packages/node_modules/@node-red/nodes/core/network/22-websocket.html

```diff
@@ -265,7 +266,10 @@
         <label for="node-config-input-tls" data-i18n="httpin.tls-config"></label>
         <input type="text" id="node-config-input-tls">
     </div>
-
+    <div class="form-row">
+        <label for="node-config-input-subprotocol"><i class="fa fa-tag"></i> <span data-i18n="websocket.label.subprotocol"></span></label>
+        <input id="node-config-input-subprotocol" type="text" data-i18n="[placeholder]websocket.placeholder.subprotocol">
```

Comment:
I don't see `websocket.placeholder.subprotocol` added to the message catalogs.

What did you have in mind for it? Won't don't generally do placeholders unless they add useful prompts to the user - such as the units of a value (eg 'seconds').

Reply:
Took this over 1:1 from the original PR, and agree it doesn't add value. Will remove the data-i18n property.